Source/Core/VideoCommon/VideoConfig.h

```diff
@@ -102,9 +102,9 @@ struct VideoConfig final
 
   // Utility
   bool bDumpTextures;
-  bool bHiresTextures;
-  bool bConvertHiresTextures;
-  bool bCacheHiresTextures;
+  bool bCustomTextures;
+  bool bConvertCustomTextures;  // what does it do?
```

Comment:
I thought so, I'll remove it at the time when updating VideoConfig.cpp

I'm also separately doing draft idea list for the terminology/semantics of the GFX gui options, to make them more technically accurate in GUI. I'll post em in a few days, but if and what would come out of it, will be after the basics of QT GUI are done, unless they decide to take my ideas early.

This option appears to be ini only, so it may not be significant, it stops a for loop in CustomTextures.cpp (HiresTextures.cpp) and I'm not that experienced yet to figure out what that actually does.